dmapiclient/data.py

```diff
@@ -181,12 +181,12 @@ def set_framework_result(self, supplier_id, framework_slug, is_on_framework, use
             user=user,
         )
 
-    def register_framework_agreement_returned(self, supplier_id, framework_slug, user, agreement_details=None):
+    def register_framework_agreement_returned(self, supplier_id, framework_slug, user, uploader_user_id=None):
```

Comment:
Should it be optional? As far as I understand, API requires the field to be set.